sdcm/nemesis.py

```diff
@@ -498,7 +506,7 @@ def disrupt_start_stop_major_compaction(self):
                              watch_for="User initiated compaction started on behalf of",
                              timeout=timeout,
                              stop_func=compaction_ops.stop_major_compaction)
-        self.tester.wait_no_compactions_running()
+        self._wait_no_compactions_running()
```

Comment:
see PR https://github.com/scylladb/scylla-cluster-tests/pull/4327 - this will be removed soon
@KnifeyMoloko can tell more.

Reply:
I am not sure what should I do?